h2o-core/src/main/java/hex/leaderboard/Leaderboard.java

```diff
@@ -415,8 +447,9 @@ public void addModels(final Key<Model>[] modelKeys) {
     }, null);
 
     if (oldLeaderKey == null || !oldLeaderKey.equals(_model_keys[0])) {
-      eventLog().info(Stage.ModelTraining,
-              "New leader: "+_model_keys[0]+", "+ _sort_metric +": "+ _metric_values.get(_sort_metric)[0]);
```

Comment:
this is the kind of log that we definitely need to raise to the client when running AutoML